include/stdlib.h

```diff
@@ -0,0 +1,26 @@
+#ifndef __SIMPLELIB_STDLIB_H
+#define __SIMPLELIB_STDLIB_H
+
+/**
+ * Not at all a full implementation of string.h,
+ * just the parts of it supported by simplelib
+ * 
+ * requirements: -
+ * c-file: stdlib.c
+ */
```

Comment:
This seems accidentally copied over :)

Reply:
copied intentionally, but forgot to update filename